core/templates/components/site-search/site-search.twig

```diff
@@ -0,0 +1,27 @@
+{#
+/**
+ * @file
+ * Global Site Searach Component
+ *
+ * A default link component.
+ *
+ * Available variables:
+ * - action: Action of the form field
+ * - method: Method of the form field
+ * - input_attributes: Additional attributes for the input field
+ * - placeholder: Placeholder text for the input field
+ * - button_attributes: Additional button attributes
+ * - search_icon_src: The href property of the icon image
+ * - additional_fields: Any additional fields, e.g., hidden input fields
+ *
+ */
+#}
+<div class="su-site-search" role="search">
+ <form action="{{ action }}" method="{{ method }}" accept-charset="UTF-8">
+  <label class="su-site-search__sr-label" for="search-field">Search this site</label>
```

Comment:
```suggestion
  <label class="su-site-search__sr-label" for="search-field">{{ search_label|default('Search this site') }}</label>
```

Reply:
I like the option of being able to change this field as well. Not allowing this will likely lead to someone doing wacky things with Twig includes or copy and paste with html.

Reply:
That is true. Applied your suggestion and will make other changes soon. Thanks!